interface/ten_ai_base/tts2.py

```
@@ -344,7 +341,8 @@ async def send_char_audio_metrics(self, request_id: str = ""):
         await self.send_usage_metrics(request_id)
 
     # send when tts audio end
-    async def send_usage_metrics(self, request_id: str = ""):
+    async def send_usage_metrics(self, request_id: str = "", extra_metadata: dict = None):
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)